src/index.html

```diff
@@ -8,7 +8,205 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
```

Comment:
I really like Chris' homework. Apart from all the good things, just a few things in general need to improve:
1) maybe try more semantic html tags like `nav`, `header`, `section`, `footer` and so on
2) you don't need to add id or class to every tag in the code, you can use the one class/id to target the child elements underneath
3) try to simplify your html structure to optimise the performance

Overall this is very solid piece of code. Well done!